prog/vm/nexus.rb

```diff
@@ -329,6 +330,11 @@ def write_params_json
       hop_update_spdk_dependency
     end
 
+    when_restart_set? do
```

Comment:
We can only restart if the vm is in wait state. I think it's fine for vms. if we had more sophisticated unavailability checks it might be desirable to still be able to restart if it's in unavailable state.

Reply:
Yes, when we have more confidence, we can do it. When we do it, we can push to `restart` label similar to the PG restart. So when it exited from the restart, it will return the state that restart initiated.